db/seeds.rb

```diff
@@ -46,3 +39,24 @@
 Admin.create(email: "admin@admin.com", password: "test12345", full_name: "admin", username: "admin" )
 puts "Admin has been created"
 #END user seed#
+
+#BEGIN portfolio seed
+
+#BEGIN stock market seed#
+Market.destroy_all
+  client = IEX::Api::Client.new(
+    publishable_token: Rails.application.credentials.iex_global_api[:publishable_token],
+    secret_token: Rails.application.credentials.iex_global_api[:secret_token],
+    endpoint: 'https://sandbox.iexapis.com/v1'
+    )
+
+  file = File.open('app/api/stock_lists/market_symbol.txt')
+  file_data = file.readlines.map(&:chomp)
+
+  file_data.each do |data|
+    Market.create(market_symbol: data, curr_price: client.price(data), logo_url: client.logo(data))
+    rescue StandardError
+      nil
+    end
+  p "populated database :)"
+#END Stock market seed#
```

Comment:
add new line at the end of file

Reply:
Kindly create a service object for your seed to have better code readability.

Reply:
Alright po, can I do this on my next PR?

Reply:
Alright!

Reply:
I'll just do this @mumble-07 so concentrate on user stories nalang